packages/circuit-ui/components/NotificationInline/NotificationInline.docs.mdx

```diff
@@ -0,0 +1,35 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+import { NotificationToast } from '@sumup/circuit-ui';
+
+# NotificationInline
+
+<Status.Stable />
+
+<Story id="notification-notificationinline--base" />
+<Props />
+
+The Notification Inline component non-disruptively provides quick and contextual inline notification and it is integrated within the content section.
+Inline notifications are frequently used to provide additional guidance to the user, like positive feedback, confirmation about an action or errors in forms, cards or dialog boxes.
+
+## Component variations
+
+### Notification Inline variants
+
+<Story id="notification-notificationinline--variants" />
+
+- Use the 'Info' variant for informational, neutral messages.
+- Use the 'Confirm' variant the successful messages.
+- Use the 'Notify' variant for warning messages, and other information a user should be aware of.
+- Use the 'Alert' variant for error messages. Be descriptive and give users clear next steps.
+
+### Inline with headline and action button
+
+<Story id="notification-notificationinline--dismissable" />
```

Comment:
Right now this is kind of mixing up the optional headline/action (name of the subsection) with the "dismissable" use case (name of the story).

Would it be clearer if we split this "component variations" section into three parts?

- `##` Component variations
  - `###` Notification variants
    - variants story
    - text about when to use each variant
  - `###` With headline and action
    - "with headline and action" story (👈 similar to the current "Dismissable" but doesn't need to be dismissable to avoid showing too many things at once)
    - text about using the optional props
  - `###` Dismissable
    - dismissable story (👈 similar to the current "Dismissable" but doesn't need to have a headline and action to avoid showing too many things at once)
    - text about adding `onClose` _and_ a `closeButtonLabel` to make the notification optionally dismissable

WDYT?

Reply:
Sounds good. I like the suggestion.